src/Xamarin.Android.Build.Tasks/Tasks/ResolveSdksTask.cs

```diff
@@ -178,7 +181,13 @@ public bool RunTask ()
 				}
 			}
 
-			foreach (var dir in MonoAndroidHelper.AndroidSdk.GetBuildToolsPaths (AndroidSdkBuildToolsVersion)) {
+			Version lintVersion = new Version ();
+			if (!string.IsNullOrEmpty (LintToolPath)) {
+				lintVersion = GetLintVersion (Path.Combine (LintToolPath, Lint));
```

Comment:
Do we actually want to do the lint version check in `<ResolveSdks/>`? We won't always need to run `lint`, but we always run `<ResolveSdks/>`, and `lint --version` takes ~0.25s for me.

I think this should be moved to the `<Lint/>` task.